### DISCUSSION TITLE
Timestamps in Elastic data

### DISCUSSION THREAD

**Discussion by regulartim** [telekom-security/tpotce#1716] [Q&A] [answered]
Hi! :) I am currently working on [GreedyBear](https://github.com/intelowlproject/GreedyBear/) and have a question regarding timestamps in the Elastic data. It seems to me that every honeypot interaction has an `@timestamp` field that is not NULL. Is this assumption correct? If so, can you tell me since which version of T-Pot this is the case?

**Comment by t3chn0m4g3** [comment, marked as answer]
Awesome to hear GreedyBear development continues!

T-Pot events are always time based events, so there are the fields `@timestamp` and `timestamp` which should never be `NULL`, but contain a UTC derived timestamp.

**Reply by regulartim** [reply]
And it always has been like that?

**Reply by t3chn0m4g3** [reply]
I sure hope so, otherwise the time series based visualizations in Kibana would not work 😅
But this is a really deep stroll into the ELK woods, so no guarantees to the all-mighty-breaking-changes 😇

**Reply by regulartim** [reply]
Alright, thank you! :)
